api/src/API/Controllers/RacesController.cs

```diff
@@ -1,12 +1,14 @@
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Logging;
 using RaceResults.Common.Models;
 using RaceResults.Data.Core;
 
 namespace RaceResults.Api.Controllers
 {
+    [Authorize]
```

Comment:
for testing, let's remove this. this is public data

Reply:
Sounds good.